packages/kuma-gui/src/app/x/components/x-empty-state/XEmptyState.vue

```diff
@@ -18,59 +18,67 @@
         ]"
         :key="title"
       >
-        <KEmptyState
-          class="x-empty-state"
+        <EntityEmptyState
+          :title="title"
+          appearance="secondary"
           data-testid="empty-block"
         >
-          <template
-            #icon
-          >
-            <slot name="icon" />
+          <template #image>
+            <div class="empty-state-icon">
+              <slot
+                v-if="slots.icon"
+                name="icon"
+              />
+              <component
+                :is="iconMapping[type]"
+                v-else-if="type && iconMapping[type]"
+                :color="KUI_COLOR_TEXT_DECORATIVE_AQUA"
+                :size="KUI_ICON_SIZE_50"
+              />
+              <AnalyticsIcon v-else />
```

Comment:
Does this need the same color/size applied?

I'm guessing this is why we see this black-on-blue-bg (but I could be wrong):

![Screenshot 2025-02-21 at 15 17 53](https://github.com/user-attachments/assets/d4b08815-8a63-404c-a15b-5503ce16017f)

Reply:
updated